client/src/components/StatsList/StatsList.tsx

```diff
@@ -0,0 +1,27 @@
+import { Stat } from "../../types/card/card.types";
+import { StatButton } from "./StatButton";
+
+type StatsListProps = {
+  stats: Stat[];
+  playerId: string;
+  playRound: () => void;
```

Comment:
As with the comment above, I think it would be better to call this prop something like "handleStatChosen", as that's what it does as far as this component is concerned. It doesn't know if we're passing it a function to play the round or do something else, it just knows a stat has been chosen by the user clicking on it.

The type definition will also need to specify that it should be passed a function that takes an stat: Stat, and playerId: string, because that's the type of function StatsButton takes.

So ```(stat: Stat, playerId: string) => void;``` same as on StatButton

Reply:
Make sense to have "handleStatChosen" prop to indicate what it does in the component. I have now changed this prop name everywhere in the StatsList folder. Thanks!